xml/s4s_about.xml

```diff
@@ -210,6 +210,158 @@
   </para>
   <para condition="noquick">
    For more information about &ph;, see <xref linkend="sec-packagehub"/>.
+   For information about life cycle and support of modules and extensions, see
+   <xref linkend="sec-s4s-service"/>.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec-s4s-service">
+  <title>Included Services</title>
+
+  <variablelist>
+   <varlistentry>
+    <term><emphasis>Extended Service Pack Overlap Support (ESPOS)</emphasis>
+    </term>
+    <listitem>
+     <para>
+      Subscriptions for &sles4sap; include Extended Service Pack Overlap
+      Support. It extends the overlap between the support periods of two
+      consecutive service packs by three years. During this period, you
+      receive support and all relevant maintenance updates under the terms of
+      Long Term Service Pack Support (LTSS). This feature is only available
```

Comment:
"only available" is not true anymore as we have ESPOS also elsewhere. Please soften or remove the sentence